tesseract/tesseract_planning/tesseract_motion_planners/include/tesseract_motion_planners/descartes/problem_generators/default_problem_generator.h

```diff
@@ -134,7 +134,7 @@ DefaultDescartesProblemGenerator(const std::string& name,
   }
   else if (isJointWaypoint(start_waypoint) || isStateWaypoint(start_waypoint))
   {
-    const Eigen::VectorXd& position = getJointPosition(start_waypoint);
+    Eigen::VectorXd position = getJointPosition(prob->manip_fwd_kin->getJointNames(), start_waypoint);
```

Comment:
This is probably a good change. It probably won't be significant as long as your joints are already in the correct order.

Reply:
Yea is should be a onetime cost up front. If not you get weird failures and it is difficult to track down.

Reply:
Well one time as in only at setup. However it will happen in the setup of each process in your process plan. I do think we should do it here, but we have also talked about making a command language checker process that goes at the top of your taskflow to check the validity of it. It could potentially also happen there to make these checks go faster. I'd say that is a task for another day though.

Reply:
Ah, that is correct. Should not be to difficult to create a program which processes the Program and corrects the format before sending it to the planners. I will see what I can come up with.